src/Package/Impl/Publishing/Handlers/RmdPublishHandler.cs

```diff
@@ -28,7 +29,7 @@ internal sealed class RmdPublishHandler : IMarkdownFlavorPublishHandler {
             string outputFile = Path.GetFileName(outputFilePath);
             string outputFolder = Path.GetDirectoryName(outputFilePath).Replace('\\', '/');
             // Run rmarkdown::render
-            return $"\"rmarkdown::render(\'{inputFile}\', output_format=\'{format}\', output_file='{outputFile}',  output_dir='{outputFolder}')\"";
```

Comment:
Alternatively, we can try and see if file is convertible to Encoding.Default and leave it as it as if it is convertable.